micromasters/urls.py

```diff
@@ -52,6 +53,8 @@
         name='financial_aid_action'),
     url(r'^api/v0/financial_aid_course_price/(?P<program_id>[\d]+)/$',
         GetLearnerPriceForCourseView.as_view(), name='financial_aid_course_price'),
+    url(r'^api/v0/financial_aid_documents_sent/(?P<financial_aid_id>[\d]+)/$',
+        DocumentsSentView.as_view(), name='financial_aid_documents_sent'),
```

Comment:
> one idea is that you could expect the desired status to be passed to the endpoint in the request body in addition to 'date_documents_sent'
Is "desired status" referring to the actions in currently in `FinancialAidActionView` (like approving an application)?  If so, that won't work because the permissions are different between those and `date_documents_sent`

For changing the names, the functionality is currently restricted to only updating `date_documents_sent`, it feels slightly misleading for the name to not indicate that.  But I don't have a strong opinion on that, so I'll go ahead and change those names as you requested.